resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+    --network [NETWORK] \
+    --asn [GOOGLE_ASN]
+
+The command output should look similar to the following example:
+
+    gcloud compute routers create router-a \
+    --region us-central1 \
+    --network network-a \
+    --asn 65001
+    
+### Create an External VPN Gateway resource
+
+Create an external VPN gateway resource that provides information to GCP about your peer VPN gateway or gateways. Depending on the HA recommendations for your peer VPN gateway, you can create external VPN gateway resource for the following different types of on-premises VPN gateways:
+
+1. Two separate peer VPN gateway devices where the two devices are redundant with each other and each device has its own public IP address.
+1. A single peer VPN gateway that uses two separate interfaces, each with its own public IP address. For this kind of peer gateway, you can create a single external VPN gateway with two interfaces.
+1. A single peer VPN gateway with a single public IP address.
+
+This interop guide covers option 2 only. 
+
+#### Create an External VPN Gateway resource for a single peer VPN gateway with two separate interfaces
+
+    gcloud compute external-vpn-gateways create [PEER_GW_NAME] \
+    --interfaces 0=[ON_PREM_GW_IP_0],1=[ON_PREM_GW_IP_1] \
+
+The command output should look similar to the following example:
+ 
+    gcloud compute external-vpn-gateways create peer-gw   \
+     --interfaces 0=204.237.220.4,1=204.237.220.35 \
+
+### Create two VPN tunnels, one for each interface on the HA VPN gateway
+
+    gcloud compute vpn-tunnels create [TUNNEL_NAME_IF0] \
+    --peer-external-gateway [PEER_GW_NAME] \
+    --peer-external-gateway-interface [PEER_EXT_GW_IF0]  \
+    --region [REGION] \
+    --ike-version [IKE_VERS] \
+    --shared-secret [SHARED_SECRET] \
+    --router [ROUTER_NAME] \
+    --vpn-gateway [GW_NAME] \
+    --interface [INT_NUM_0]
+    
+    gcloud compute vpn-tunnels create [TUNNEL_NAME_IF1] \
+    --peer-external-gateway [PEER_GW_NAME] \
+    --peer-external-gateway-interface [PEER_EXT_GW_IF1] \
+    --region [REGION] \
+    --ike-version [IKE_VERS] \
+    --shared-secret [SHARED_SECRET] \
+    --router [ROUTER_NAME] \
+    --vpn-gateway [GW_NAME] \
+    --interface [INT_NUM_1]
+    
+The command output should look similar to the following example:
+ 
+    gcloud compute vpn-tunnels create tunnel-a-to-on-prem-if-0 \
+    --peer-external-gateway peer-gw \
+    --peer-external-gateway-interface 0  \
+    --region us-central1 \
+    --ike-version 2 \
+    --shared-secret mysharedsecret \
+    --router router-a \
+    --vpn-gateway ha-vpn-gw-a \
+    --interface 0
+    
+    gcloud compute vpn-tunnels create tunnel-a-to-on-prem-if-1 \
+    --peer-external-gateway peer-gw \
+    --peer-external-gateway-interface 1  \
+    --region us-central1 \
+    --ike-version 2 \
+    --shared-secret mysharedsecret \
+    --router router-a \
+    --vpn-gateway ha-vpn-gw-a \
+    --interface 1
+    
+### Create Cloud Router interfaces and BGP peers
```

Comment:
The step numbering in this section is showing up as "1." for every step.  I suspect that's due to Markdown indentation or spacing issues.